modules/ingress/README.md

```diff
@@ -40,20 +43,28 @@ module "ingress" {
   alb_name = "test"
   hostname = "test.devops.dasmeta.com"
   annotations = {
-       "alb.ingress.kubernetes.io/load-balancer-name" = "test-ingress"
-       "alb.ingress.kubernetes.io/scheme"             = "internet-facing"
-       "alb.ingress.kubernetes.io/subnets"            = "subnet-0ebc13842a5f, subnet-05b72a3769b, subnet-0da2ece4bb4229"
-       "alb.ingress.kubernetes.io/backend-protocol"   = "HTTP"
-       "alb.ingress.kubernetes.io/certificate-arn"    = "arn:aws:acm:us-east-1:5********68:certificate/a55ee6eb****1706"
-       "alb.ingress.kubernetes.io/group.name"         = "dev"
-       "alb.ingress.kubernetes.io/listen-ports"       = "[{\"HTTPS\":443}, {\"HTTPS\":80}]"
-       "kubernetes.io/ingress.class"                  = "alb"
+       "alb.ingress.kubernetes.io/load-balancer-name"   = "test-ingress"
```

Comment:
I suggest we move those into separate variables and merge inside the module.
When using the module I don't need to understand how this is constructed and also if we refactor the module and use some other mechanism that will force module signature update when ingress module should be implementation agnostic.